### PR TITLE
Added a comparison of manual implementation of KL with implementations based on tf.contrib.distributions

### PR DESCRIPTION
**Pull request by vincentadam87**
implementations by @vdutor 
tests by @vincentadam87

### CODE REVIEW

**Comment by vdutor** [comment]
I would propose to change the upstream to `vdutor/multiple-output-gps` instead of directly merging into `master`. The functionality you added to the KLs is used by the multioutput framework and I need it to fix a currently broken unit test.

File: tests/test_kldiv.py [hunk 1]
```diff
@@ -24,20 +25,62 @@
 from gpflow import settings
 from gpflow.test_util import session_tf
 
+
+def gauss_kl_tf_distributions(q_mu, q_sqrt, K=None):
+    """
+    Kullbach-Leiber divergence KL[q(U) || p(U)],
```
Comment (st--) [inline tests/test_kldiv.py]: Kullbach-Leiber -> Kullback-Leibler

File: tests/test_kldiv.py [hunk 2]
```diff
@@ -24,20 +25,62 @@
...
+    with q(U) ~ N(q_mu, q_sqrt^2) the variational Gaussian posterior
+    and p(U) ~ N(0, K) the prior. If K is None we assume a whitened
+    prior, p(U) ~ N(0, I).
+
+    We assume L independent distributions, then
+
+    :param q_mu: L variational means, M x L
+    :param q_sqrt: L variational covariances,
+                - cholesky: L x M x M or
+                - diag elements: M x L
```
Comment (st--) [inline tests/test_kldiv.py]: just for clarification - is this still the sqrt of the diagonal elements?

**Comment by st--** [comment]
@vdutor should we just merge this ?

**Comment by vdutor** [comment]
Yes, this can be merged.

**Pull request closed without merging**